libs/coin-tester/src/docker.ts

```diff
@@ -0,0 +1,96 @@
+import path from "path";
+import axios from "axios";
+import fs from "fs/promises";
+import { v2 as compose } from "docker-compose";
+import SpeculosTransportHttp from "@ledgerhq/hw-transport-node-speculos-http";
+import { ENV } from "./types";
+import chalk from "chalk";
+import { SignOperationEvent } from "@ledgerhq/types-live";
+
+const { API_PORT } = process.env as ENV;
+const cwd = path.join(__dirname);
+
+const delay = (timing: number) => new Promise(resolve => setTimeout(resolve, timing));
+
+export const spawnSigner = async (
+  service: "speculos",
+  nanoAppEndpoint: `/${string}`,
+): Promise<{
+  transport: SpeculosTransportHttp;
+  onSignerConfirmation: (e?: SignOperationEvent) => Promise<void>;
+}> => {
+  console.log(`Starting ${service}...`);
+
+  const { data: blob } = await axios({
```

Comment:
I would use `live-network` instead of directly `axios`

Reply:
Can be discussed but I'd rather not as it was though for Ledger Live